### PR TITLE
Basic Info GA Events API Automation

### PR DESCRIPTION
**Pull request by sangeetha-nayak**
PRESS2-1368 : Cypress Test Events API: Basic Info

### CODE REVIEW

File: tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // <reference types="Cypress" />
 import {
 	DrawerActivityForMenu,
-	DrawerClose
+	DrawerClose,
```
Comment (avneet-raj) [inline tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js:4]: Do we need **,** here?

File: tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js [hunk 2]
```diff
@@ -205,4 +211,24 @@ describe( 'Basic Info Page', function () {
...
+	it( 'Basic Info Events API', () => {
+		const label_keys = [
+			'',
+			'title',
+			'tagline',
+			'logo_added',
+			'platform',
+			'platform',
+		];
+		const actual_values = [ '', title, desc, '', 'facebook', 'instagram' ];
```
Comment (avneet-raj) [inline tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js:224]: Are we checking only for 2 social links?

File: tests/cypress/integration/wp-module-support/drawer.cy.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
 // <reference types="Cypress" />
+const customCommandTimeout = 30000;
 
 export const CheckDrawerDisabled = () => {
-	cy.get( '.nfd-onboarding-drawer__toggle-button' )
+	cy.get( '.nfd-onboarding-drawer__toggle-button', {
+		timeout: customCommandTimeout,
```
Comment (avneet-raj) [inline tests/cypress/integration/wp-module-support/drawer.cy.js:6]: Why do we need this to be added? Will it not break the existing TCs?

File: tests/cypress/integration/wp-module-support/drawer.cy.js [hunk 4]
```diff
@@ -65,7 +68,9 @@ export const DrawerActivityForSubMenu = (
 	cy.get( '.nfd-onboarding-drawer__panel-inner' )
 		.scrollIntoView()
 		.should( 'be.visible' );
-	cy.get( '.nfd-onboarding-drawer__panel-back' )
+	cy.get( '.nfd-onboarding-drawer__panel-back', {
+		timeout: customCommandTimeout,
```
Comment (avneet-raj) [inline tests/cypress/integration/wp-module-support/drawer.cy.js:72]: Same here?

File: tests/cypress/integration/wp-module-support/EventsApi.cy.js [hunk 5]
```diff
@@ -1,76 +1,99 @@
...
+		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=wp-setup&_locale=user',
+	site_primary:
+		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=wp-setup&_locale=user',
+	site_primary_ecomm:
+		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=ecommerce&_locale=user',
+	site_secondary:
+		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=wp-setup&_locale=user',
+	site_secondary_ecomm:
+		'/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=ecommerce&_locale=user',
+	basic_info_batch:
```
Comment (avneet-raj) [inline tests/cypress/integration/wp-module-support/EventsApi.cy.js:18]: Let's keep it as only 'basic_info' as we did for other pages

File: tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js [hunk 6]
```diff
@@ -205,4 +211,24 @@ describe( 'Basic Info Page', function () {
...
+
+		cy.wait( 2000 );
+
```
Comment (avneet-raj) [inline tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js:227]: Let's remove these extra spaces as well.

File: tests/cypress/integration/wp-module-support/drawer.cy.js (hunk 3 above)
Comment (sangeetha-nayak) [reply tests/cypress/integration/wp-module-support/drawer.cy.js:6]: It just increases wait timeout from 5000(cypress default) to 30000. Its not going to overwrite any existing config and break other cases.

File: tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js (hunk 6 above)
Comment (sangeetha-nayak) [reply tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js:258]: Done

File: tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js (hunk 1 above)
Comment (sangeetha-nayak) [reply tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js]: Removed it.

File: tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js (hunk 2 above)
Comment (sangeetha-nayak) [reply tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js]: Added for all

File: tests/cypress/integration/wp-module-support/EventsApi.cy.js (hunk 5 above)
Comment (sangeetha-nayak) [reply tests/cypress/integration/wp-module-support/EventsApi.cy.js]: Done

**Review by avneet-raj — APPROVED** [review]

**Pull request merged**